subprojects/launcher/src/main/java/org/gradle/launcher/daemon/client/DaemonClient.java

```diff
@@ -154,6 +158,15 @@ public BuildActionResult execute(BuildAction action, BuildActionParameters param
             }
         }
 
+        if (newDaemon != null) {
+            throw new NoUsableDaemonFoundException("A new daemon started but cannot be connected. " +
+                "This might indicate that the networking configuration blocks the connection to the daemon. " +
+                "Details of the created daemon: " +
```

Comment:
Would it be possible to link to a page in the user manual with troubleshooting steps?

Reply:
I've created a new section on the troubleshooting page and linked it in the exception.